src/main/java/com/imagination/cbs/controller/ReasonsForRecruitingController.java

```diff
@@ -0,0 +1,24 @@
+package com.imagination.cbs.controller;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.imagination.cbs.dto.ReasonsForRecruitingDto;
+import com.imagination.cbs.service.impl.ReasonsForRecruitingServiceImpl;
+
+@RestController
+@RequestMapping("/reasonsForRecruiting")
+public class ReasonsForRecruitingController {
+	
+	@Autowired
+	private ReasonsForRecruitingServiceImpl reasonsForRecruitingServiceImpl;
```

Comment:
if we are injecting implementation directly here then why do we need interface ? also just a suggestion ... we have interfaces for all the services ... we dont need interface until we have multiple implementations ....creating interfaces and implementing it in one single class and no where else is old approach.